src/chrome/content/rules/ccxmedia.org.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="ccxmedia.org">
+	<target host="ccxmedia.org" />
+	<target host="www.ccxmedia.org" />
+	<target host="store.ccxmedia.org" />
+
+	<rule from="^http://www\.ccxmedia\.org/" to="https://ccxmedia.org/" />
```

Comment:
This line is superfluous: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects